modules/300-prometheus/templates/alertmanager/internal/alertmanager.yaml

```diff
@@ -71,6 +71,9 @@ spec:
             alertmanager: {{ .name }}
         topologyKey: kubernetes.io/hostname
   {{- end }}
+{{- if .Values.global.modules.publicDomainTemplate }}
+  externalUrl: {{ include "helm_lib_module_uri_scheme" . }}://{{ include "helm_lib_module_public_domain" (list . "grafana") }}/alertmanager/monitoring
```

Comment:
```suggestion
  externalUrl: {{ include "helm_lib_module_uri_scheme" . }}://{{ include "helm_lib_module_public_domain" (list . "grafana") }}/alertmanager/{{ .name }}
```

Reply:
Thank you very much. I didn't check the path in the Ingress manifest. I've fixed everything now.